versioneer.py

```diff
@@ -1769,12 +1768,12 @@ def do_setup():
     manifest_in = os.path.join(root, "MANIFEST.in")
     simple_includes = set()
     try:
-        with open(manifest_in, "r") as f:
+        with open(manifest_in) as f:
             for line in f:
                 if line.startswith("include "):
                     for include in line.split()[1:]:
                         simple_includes.add(include)
-    except EnvironmentError:
+    except OSError:
```

Comment:
Change of error from EnvironmentError to OSError. According to the [Built-in Exceptions docs](https://docs.python.org/3/library/exceptions.html#OSError.filename2), EnvironmentError was merged into OSError in 3.3